modules/aws_ecs/main.tf

```diff
@@ -2,15 +2,11 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 5.0"
     }
   }
 }
 
-provider "aws" {
-  region = var.aws_region
```

Comment:
is this not needed anymore?

Reply:
It's typically best practice to not have the provider block defined in a module.  Defining the block in the module can cause issues when trying to delete or comment out that module.

Reply:
i see! would changing this break existing usage?

Reply:
it shouldn't, but i can test to be sure

Reply:
tested various scenarios, did not break existing usage